lib/segment/src/vector_storage/sparse/simple_sparse_vector_storage.rs

```diff
@@ -179,7 +179,7 @@ impl VectorStorage for SimpleSparseVectorStorage {
     }
 
     fn is_on_disk(&self) -> bool {
-        true
+        false
```

Comment:
I think the `SimpleSparseVectorStorage` is on disk.
The data is only managed by RocksDB without in memory replication.

Reply:
Fixed, Thanks!